apps/src/applab/DesignModeHeaders.jsx

```diff
@@ -56,25 +60,42 @@ export default React.createClass({
       </span>
     );
 
+    var isRtl = this.props.localeDirection === 'rtl';
+    const hasFocus = !this.props.isRunning;
+
     return (
-      <div id="design-headers">
-        <div id="design-toolbox-header" className="workspace-header" style={styles.toolboxHeader}>
+      <PaneHeader
+        id="design-headers"
+        dir={this.props.localeDirection}
+        hasFocus={hasFocus}
+        style={{color: 'white'}}
+      >
+        <PaneSection id="design-toolbox-header" className="workspace-header" style={styles.toolboxHeader}>
           {manageAssetsIcon}
           <span>{applabMsg.designToolboxHeader()}</span>
           <span className="workspace-header-clickable" onClick={this.onToggleToolbox}>&nbsp;{msg.hideToolbox()}</span>
-        </div>
-        <div
+        </PaneSection>
+        <PaneSection
           className="workspace-header"
           onClick={this.onToggleToolbox}
           style={styles.showToolboxHeader}
         >
           <span className="workspace-header-clickable">{msg.showToolbox()}</span>
           {manageAssetsIcon}
-        </div>
-        <div id="design-workspace-header" className="workspace-header">
+        </PaneSection>
+        <PaneButton
+          id="design-mode-versions-header"
+          iconClass="fa fa-clock-o"
+          label={msg.showVersionsHeader()}
+          headerHasFocus={hasFocus}
+          isRtl={isRtl}
+          onClick={() => this.props.handleVersionHistory()}
```

Comment:
I started doing this to prevent the event object from being passed to the handler since it caused a problem in another situation, but it seems harmless here. I've removed the wrappers.